CMakeLists.txt

```diff
@@ -17,12 +17,14 @@ cmake_policy(SET CMP0025 NEW)
 cmake_policy(SET CMP0048 NEW)
 cmake_policy(SET CMP0057 NEW)
 
-project(Umpire 
+project(Umpire
   LANGUAGES CXX C
   VERSION 0.3.1)
 
 set(ENABLE_OPENMP On CACHE Bool "")
 set(ENABLE_CUDA On CACHE Bool "")
+option(ENABLE_NUMA_HOST "Add allocators for host NUMA nodes" Off)
```

Comment:
I think that the options should be:

- `ENABLE_NUMA`: host NUMA
- `ENABLE_NUMA_DEVICES`: GPU numa nodes

Sorry for flip-flopping a bit here.

Reply:
No worries, I was the one who made this initial structure. I'll go ahead and make the change.